docs/troubleshooting/handling-node-failures.rst

```diff
@@ -102,53 +94,82 @@ in the past and then had Raft enabled, and to clusters that were bootstrapped us
   to life and communicate with the rest of the cluster, setup firewall rules or otherwise 
   isolate your alive nodes to reject any communication attempts from these dead nodes.
 
-* Prepare your service for downtime before proceeding.
-  Entering ``RECOVERY`` mode requires a node restart. Restarting an additional node while 
-  some nodes are already dead may lead to unavailability of data queries (assuming that 
-  you haven't lost it already). For example, if you're using the standard RF=3, 
-  CL=QUORUM setup, and you're recovering from a stuck upgrade procedure because one 
-  of your nodes is dead, restarting another node will cause temporary data query 
-  unavailability (until the node finishes restarting). 
+* Ensure all live nodes are in the normal state using
+  :doc:`nodetool status </operating-scylla/nodetool-commands/status>`. If there is a node
+  that is joining or leaving, it cannot be recovered. You must permanently stop it and remove
+  during the manual recovery procedure.
```

Comment:
note: I realized there is a lie here. Removing the node may be necessary **or not** depending on the timing when quorum was lost. The node is always unrecoverable anyway. I'll send a fix for it (I can't do it now).

Reply:
fixed